assets/stylesheets/back_to_top.css

```diff
@@ -0,0 +1,8 @@
+.back-to-top {
+    bottom: 20px;
+    cursor: pointer;
+    display: none;
+    position: fixed;
+    right: 20px;
+    z-index: 1000;
+}
```

Comment:
I'd also add `.back-to-top i { padding-right: initial; }`. Otherwise the icon isn't centered correctly within the button.

Reply:
Doesn't look like it changes anything.

Reply:
With Bootstrap defaults the icon is perfectly centered, must be one of the custom settings openQA uses.

Reply:
It's not properly in the center.